src/io.cpp

```diff
@@ -173,10 +173,10 @@ Point FormatPQR::load(const std::string &file, FormatPQR::Tpvec &p, bool keep_ch
                             log_charge = false;
                         }
                         if (keep_charges)
-                            faunus_logger->info("charge mismatch on atom {0} {1}: using {2} (PQR) over {3} (atomlist)",
+                            faunus_logger->debug("charge mismatch on atom {0} {1}: using {2} (PQR) over {3} (atomlist)",
```

Comment:
I think this information should be printed by default. If two different charges are given, the user may have different intentions.

Reply:
It works like that: If there is a first charge mismatch in the moleculeʼs topology a warning for the molecule is raised. All the mismatches, atom by atom, are then printed at the debug level. The change was motivated by Marcoʼs simulation of a protein with hundreds lines at the info level which looked confusing.

Reply:
Ahh I see. In any case Marco should fix his input :-)